src/views/catalog/CreateFromInstanceTypes/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/components/ComposableDrilldownSelect/ComposableDrilldownSelect.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {
```

Comment:
what do u think about making a hook that will be more generic so we can use useClickedOutside ? so other components in the future will be easy to use in this case? or searching for npm package that already implement this?

Reply:
I can extract it to a general location, I also found a package that includes this hook with a few more hooks that might help in the future, for now since this is really small I'll simply extract it to a util hook

Reply:
I extracted it for now under the select, in part 2 I might this functionality in some other place, than I would put it in utils